doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -188,25 +196,25 @@ int main(int argc, char** argv)
   context->solve(res);
   if (res.error_code_.val != res.error_code_.SUCCESS)
   {
-    ROS_ERROR("Could not compute plan successfully");
+    RCLCPP_ERROR(LOGGER, "Could not compute plan successfully");
     return 0;
   }
 
   // Visualize the result
   // ^^^^^^^^^^^^^^^^^^^^
-  ros::Publisher display_publisher =
-      node_handle.advertise<moveit_msgs::DisplayTrajectory>("/display_planned_path", 1, true);
-  moveit_msgs::DisplayTrajectory display_trajectory;
+  auto display_publisher = motion_planning_api_tutorial_node->create_publisher<moveit_msgs::msg::DisplayTrajectory>(
```

Comment:
nit: I prefer not to use `auto` in tutorials so people can look up the type easily. If it's a ridiculously long type, then OK, this is 100% fine.

Reply:
Fixed.